app/views/podcast_planner/_form_draft.html.erb

```diff
@@ -55,7 +55,7 @@
             </button>
           </div>
 
-          <% 24.times do |i| %>
+          <% (12 * PodcastPlanner::MAX_YEARS).times do |i| %>
```

Comment:
Hmm... is this off by 1?  I'm now getting months out to May 2027 here.  But the dropdown field only goes out to May 14th 2026.

Reply:
the "start" date should go out 2 years, but then this calendar and the validations allow setting episodes 3 days out.
this mirrors how it worked before, where there was a year differential between the last start date, and the end of where episodes can be created out until.

Reply:
Yeah, so `past_max_date?` is going to be May 2027.  Does that mean the `Start Date` field should also go out to 2027?

Reply:
...unless I set it up wrong, but that was what I was going for.

Reply:
should the start date and end date match up? they didn't before - I believe they were a year apart before, and that was what I tried to retain here, just shifting both from 1 and 2 years, to 2 and 3 years

Reply:
I think the dropdown-calendar and the months at the bottom should be the same.

In other words ... you're allowed to `Start Date` your planned episodes anytime between tomorrow and 3 years from now.  BUT - that `past_max_date?` method enforces a hard limit of 3 years from now.

(If I start planning weekly episodes 2 years and 11 months from now... it'll only give me 4 no matter how many I ask for).

Reply:
done